cmd/lncli/cmd_version.go

```diff
@@ -0,0 +1,52 @@
+package main
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/lightningnetwork/lnd/build"
+	"github.com/lightningnetwork/lnd/lnrpc/verrpc"
+	"github.com/urfave/cli"
+)
+
+var versionCommand = cli.Command{
+	Name:     "version",
+	Category: "Info",
+	Usage:    "Display deamon version info.",
+	Description: `
+	Returns version information about the running daemon.
+	`,
+	Action: actionDecorator(version),
+}
+
+func version(ctx *cli.Context) error {
+	conn := getClientConn(ctx, false)
+	defer conn.Close()
+
+	versions := &verrpc.CLIVersionDisplay{
```

Comment:
another alternative is to make an `lnclirpc` package that is just used for custom display messages. i slightly favor that way since it doesn't pollute the protos with lncli-specific display structs, wdyt?

Reply:
latest version creates a `lnrpc/lnclipb` package which is nice imo bc it doesn't pollute the `verrpc` namespace.